File: client/Client/Network/Messages/Session/DeselectSectionMessage.cs
namespace LoadRunnerClient.Messages
{
    public class DeselectSectionMessage
    {
        public const string TYPE = "DeselectSectionMessage";
        private int _section;
        public int section { get => _section; set => _section = value; }

        public DeselectSectionMessage()
        {
        }

        public DeselectSectionMessage(int sec)
        {
            this.section = sec;
        }
    }
}

File: server/src/main/java/de/hsrm/mi/swtpro03/server/network/messages/session/DeselectSectionMessage.java
package de.hsrm.mi.swtpro03.server.network.messages.session;

public class DeselectSectionMessage {
	public static final String TYPE = "DeselectSectionMessage";
	private int section;

	public DeselectSectionMessage() {
	}

	public DeselectSectionMessage(int x) {
		this.section = x;
	}

	public int getSection() {
		return section;
	}

	public void setSection(int section) {
		this.section = section;
	}
}
